docs/modules/ROOT/pages/tutorials/getting-started/java-terminal.adoc

```diff
@@ -0,0 +1,310 @@
++
+[source,shell]
+--
+$ oc -n fortune-java apply -f deployment.yaml
+deployment.apps/fortune-java created
+service/fortune-java created
+$ oc -n fortune-java get pods --watch
+NAME                         READY   STATUS    RESTARTS   AGE
+fortune-java-6fbd5484cf-k47gt   1/1     Running   0          11s
+--
+
+== Step 2: Publish your Application
+
+At the moment your container is running but it's not available from the Internet. To be able to access our application, we must create an `Ingress` object.
+
+. Create another file called `ingress.yaml` with the following contents, customizing the parts marked as `[YOUR_APP_NAME]` and `[YOUR_PREFERRED_ZONE]` to your liking:
++
+[source,yaml]
+--
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-production
+  name: fortune-java-ingress
+  namespace: fortune-java # <1>
+spec:
+  rules:
+  - host: [YOUR_APP_NAME].apps.[YOUR_PREFERRED_ZONE].appuio.cloud # <2>
+    http:
+      paths:
+      - pathType: Prefix
+        path: /
+        backend:
+          service:
+            name: fortune-java
+            port:
+              number: 8080
+  tls:
+  - hosts:
+    - [YOUR_APP_NAME].apps.[YOUR_PREFERRED_ZONE].appuio.cloud
+    secretName: fortune-java-cert
+--
+<1> Make sure this annotation matches exactly the name of your project: `fortune-java`
+<2> Replace the placeholders `YOUR_APP_NAME` and `YOUR_PREFERRED_ZONE` with valid values.
+
+. Apply the ingress object to your {product} project and wait until you route shows as available.
++
+[source,shell]
+--
+$ oc -n fortune-java apply -f ingress.yaml
+ingress.networking.k8s.io/fortune-java-ingress created
+$ oc -n fortune-java get routes --watch
+NAME                      HOST/PORT                                         PATH   SERVICES    PORT    TERMINATION     WILDCARD
+fortune-java-ingress-4pk2j   fortune-java.apps.[YOUR_PREFERRED_ZONE].appuio.cloud   /      fortune-java   <all>   edge/Redirect   None
+--
+
+. After a few seconds, you should be able to get your daily fortune message using `curl`!
++
+[source,shell]
+--
+$ curl https://[YOUR_APP_NAME].apps.[YOUR_PREFERRED_ZONE].appuio.cloud --header "Accept: text/plain"
+$ curl https://[YOUR_APP_NAME].apps.[YOUR_PREFERRED_ZONE].appuio.cloud --header "Accept: application/json"
+--
+
+== Step 3: There's no Step 3!
+
+The "Fortune in  Java" application is now running on {product}. Congratulations!
+
+What's next? To run your own application written in Java or using the Quarkus Framework on {product}, follow these steps:
+
+* Containerize the application making sure it's compatible with {product}. The `Dockerfile` above can serve as a starting point.
+* Enhance the deployment for your application with liveness and health probes, or better yet, create a https://helm.sh/[Helm] chart.
+* Configure your CI/CD system to automatically deploy your application to your cluster.
```

Comment:
Same comment here like on the web version.

Reply:
Done